.github/workflows/release-pr.yml

```diff
@@ -1,14 +1,15 @@
-name: Create release PR
+name: release Project
 
 on:
+   push:
+      branches:
+         - main
+      paths:
+         - CHANGELOG.md
    workflow_dispatch:
-      inputs:
-         release:
-            description: 'Define release version (ex: v1, v2, v3)'
-            required: true
 
 jobs:
-   release-pr:
-      uses: OliverMKing/javascript-release-workflow/.github/workflows/release-pr.yml@main
+   release:
+      uses: OliverMKing/action-release-workflows/.github/workflows/release_js_project.yaml@main
```

Comment:
We should probably move the action-release workflows into an Azure repo. Also unsure about using @main instead of a sha.

Reply:
done